README.md

```diff
@@ -12,8 +12,21 @@ Light reading can be found on Wikipedia's [After Dark](https://en.wikipedia.org/
 Original foundation was written by [James Carnley](http://jamescarnley.com). Google Code [Project](https://code.google.com/p/lunatic-fringe/).
 
 ## Usage
-1. Open index.html in browser
-2. Play
+NOTE: With the refactor that separated out the code into multiple files, it is no longer possible to just open up index.html in a browser due to being blocked by CORS policy. So until I think of a way to handle this to potentially get around that without requiring that you turn off blocking for CORS violations, the best way to run this locally is to run a local server to host the files. The easiest way I have found to do this is as follows:
+1. Download Visual Studio Code (it's free) https://code.visualstudio.com/
+2. Find an extenstion for Visual Studio Code for running a local server (Live Server is one I found that has lots of downloads and a high star rating https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer&ssr=false#review-details)
+3. Open the folder containing the code on your local machine
+4. Start your local server (for the Live Server extenstion, you should be able to just click the 'Go Live' button in the bottom right corner of the Visual Studio Code window)
+5. Play
+
+## Controls
```

Comment:
Technically there are some keys that can be used outside of these (D: Toggle debug mode, A: Advance 1 frame while in debug mode) but since those are more of debugging controls I did not add them here.